src/util/serverUtils.js

```diff
@@ -0,0 +1,72 @@
+import https from 'https';
```

Comment:
yes - that change has been made in the starter repo but hasn't been carried over yet